examples/templates/docker/build/Dockerfile

```diff
@@ -1,3 +1,17 @@
 FROM ubuntu
 
-RUN apt-get update && apt-get install -y curl wget git vim golang
+RUN apt-get update \
+	&& apt-get install -y \
+	curl \
+	git \
+	golang \
+	sudo \
+	vim \
+	wget
+
+ARG USER=coder
```

Comment:
nit: I used to move all args to the top of the file, so it's easier to find.

Reply:
It's better to place it as low as possible so it doesn't cause a rebuild of previous steps when the value changes.